src/js/lib/youtube.js

```diff
@@ -65,6 +69,50 @@ export function pimpYouTubePlayer(videoId, node, height, width, chapters) {
             });
         });
     });
+
+
+    function chapterTimer(youTubePlayer, playerTotalTime) {
+        let chapterCurrentProgress;
```

Comment:
this whole block needs a bit of a refactor 😢  but not really in the scope of this PR